src/cron/cron.py

```diff
@@ -43,70 +46,75 @@
                 EMAIL,
                 PRENOM,
                 NOM,
-                USER_DOMAIN,
-                USER_FIRST_LOGIN,
-                USER_LAST_LOGIN,
-                USER_NB_DAYS_BETWEEN_FIRST_AND_LAST_LOGIN,
-                USER_INACTIVITY,
-                USER_SIRET,
-                count(DOC_ID) AS USER_NB_DOCUMENTS
+                DOMAIN,
+                FIRST_LOGIN,
+                LAST_LOGIN,
+                NB_DAYS_BETWEEN_FIRST_AND_LAST_LOGIN,
+                INACTIVITY,
+                SIRET,
+                TYPE,
+                count(DOC_ID) AS NB_DOCUMENTS
             FROM raw_brevo_data
             GROUP BY
                 EMAIL,
                 PRENOM,
                 NOM,
-                USER_DOMAIN,
-                USER_FIRST_LOGIN,
-                USER_LAST_LOGIN,
-                USER_NB_DAYS_BETWEEN_FIRST_AND_LAST_LOGIN,
-                USER_INACTIVITY,
-                USER_SIRET;
-            """)
+                DOMAIN,
+                FIRST_LOGIN,
+                LAST_LOGIN,
+                NB_DAYS_BETWEEN_FIRST_AND_LAST_LOGIN,
+                INACTIVITY,
+                SIRET,
+                TYPE;
+            """
+        )
         users = cur.fetchall()
 
 brevo_payload = {
-                    "emailBlacklist": False,
-                    "disableNotification": False,
-                    "smsBlacklist": False,
-                    "updateExistingContacts": True,
-                    "emptyContactsAttributes": False,
-                    "jsonBody":[],
-                    "listIds": ID_BREVO_LIST,
-                }
-
-brevo_attributes = [
-                    "PRENOM",
-                    "NOM",
-                    "USER_DOMAIN",
-                    ATTRS_PREFIX + "USER_FIRST_LOGIN",
-                    ATTRS_PREFIX + "USER_LAST_LOGIN",
-                    ATTRS_PREFIX + "USER_NB_DAYS_BETWEEN_FIRST_AND_LAST_LOGIN",
-                    ATTRS_PREFIX + "USER_INACTIVITY",
-                    "USER_SIRET",
-                    ATTRS_PREFIX + "USER_NB_DOCUMENTS"
-                    ]
-
-EMAIL_ATTR_INDEX = 0
-USER_FIRST_LOGIN_INDEX = 3
-USER_LAST_LOGIN_INDEX = 4
-
-def normalize_date(value: date|None) -> str|None:
-    return value.strftime('%Y-%m-%d') if value is not None else value
+    "emailBlacklist": False,
+    "disableNotification": False,
+    "smsBlacklist": False,
+    "updateExistingContacts": True,
+    "emptyContactsAttributes": False,
+    "jsonBody": [],
+    "listIds": ID_BREVO_LIST,
+}
+
+
+def user_to_brevo_attributes(user):
+    return {
+        "PRENOM": user["prenom"],
+        "NOM": user["nom"],
+        "USER_DOMAIN": user["domain"],
+        ATTRS_PREFIX + "USER_FIRST_LOGIN": user["first_login"],
+        ATTRS_PREFIX + "USER_LAST_LOGIN": user["last_login"],
+        ATTRS_PREFIX
+        + "USER_NB_DAYS_BETWEEN_FIRST_AND_LAST_LOGIN": user[
```

Comment:
Is it possible to format otherwise?
Either:
```suggestion
        ATTRS_PREFIX + "USER_NB_DAYS_BETWEEN_FIRST_AND_LAST_LOGIN": user[
```

Or:
```suggestion
        ATTRS_PREFIX
             + "USER_NB_DAYS_BETWEEN_FIRST_AND_LAST_LOGIN": user[
```

Reply:
As the code is formatted with black I would not keep this suggestion.

Reply:
Can you then add black in the `pyproject.toml` file?

https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-format

And if possible, add `black` as a dev / optional / whatever dependency? (`uv add --dev black` or whatever)

Reply:
I have also added this commit:
4621211

The idea is, as much as possible, it is clear that the lint tool is black. The user may then run `uvx black .`